packages/web/src/scss/components/RadioField/index.html

```diff
@@ -0,0 +1,21 @@
+{{#> layout }}
+  <h2 class="typography-heading-medium-text mt-1100 mb-800">RadioField</h2>
+  <div class="example-RadioField-grid">
+    <label for="radiofield1" class="RadioField">
+      <input type="radio" id="radiofield1" name="example" class="RadioField__input" />
+      <span class="RadioField__label">Radio unselected</span>
+    </label>
+    <label for="radiofield2" class="RadioField RadioField--disabled">
+      <input type="radio" id="radiofield2" name="example" class="RadioField__input" disabled />
+      <span class="RadioField__label">Radio unselected</span>
+    </label>
+    <label for="radiofield3" class="RadioField">
+      <input type="radio" id="radiofield3" name="example" class="RadioField__input" checked />
+      <span class="RadioField__label">Radio selected</span>
+    </label>
+    <label for="radiofield4" class="RadioField RadioField--disabled">
+      <input type="radio" id="radiofield4" name="example2" class="RadioField__input" checked disabled />
```

Comment:
```suggestion
      <input type="radio" id="radiofield4" name="example" class="RadioField__input" checked disabled />
```

Reply:
no, this is on purpose.. I wanted to demonstrate the disabled checked state, but you can have checked under one name only once